reference/cli/sailsdebug.md

```diff
@@ -23,6 +23,26 @@ debugger listening on port 5858
 
 > To use the standard (command-line) node debugger with sails, you can always just run `node debug app.js`.
 
+### Use node-inspector
+
+To use node-inspector to debug Sails app, install node-inspector and launch it
```

Comment:
“To debug your Sails app using node-inspector, first install and launch node-inspector”